src/Symfony/Component/Messenger/Worker.php

```diff
@@ -101,7 +101,7 @@ public function run(array $options = []): void
             }
         }
 
-        $this->dispatchEvent(new WorkerStoppedEvent($this));
+        $this->dispatchEvent(new WorkerStoppedEvent($this, $this->receivers));
```

Comment:
I'd prefer not adding a new class + argument

instead, we could add the reset loop after this call, isn't it?

Reply:
True, good idea :+1: